modules/javafx.graphics/src/main/java/javafx/scene/AccessibleRole.java

```diff
@@ -822,4 +822,16 @@ public enum AccessibleRole {
      * </ul>
      */
     TREE_VIEW,
+
+    /**
+     * Dialog role.
+     * <p>
+     * Attributes:
+     * <ul>
+     * <li> {@link AccessibleAttribute#TEXT} </li>
+     * <li> {@link AccessibleAttribute#ROLE_DESCRIPTION} </li>
```

Comment:
What is the purpose of adding `ROLE_DESCRIPTION` here? It isn't listed anywhere else except as an optional `Node` attribute.

Reply:
Reason to include `ROLE_DESCRIPTION` explicitly is that role description is fetched and read by the accessibility client application.
The change in MacAccessible.java line 1302 and in WinAccessible line 799, are specifically for ROLE_DESCRIPTION.
Windows Narrator appends this role description after dialog title.